build.gradle

```diff
@@ -41,6 +42,11 @@ test {
     useJUnitPlatform()
 }
 
+javafx {
+    version = "11.0.2"
+    modules = [ 'javafx.controls', 'javafx.fxml' ]
+}
+
```

Comment:
Why are we importing javafx again? It's already a dependency under `depedencies`